app/views/feeds/show.html.erb

```diff
@@ -9,7 +9,7 @@
   <%= tag.div id: dom_id(@feed), class: "max-w-xl mt-5 mx-auto" do %>
     <div class="my-8">
       <h4 class="block text-md font-bold text-gray-700">Email address</h4>
-      <pre class="my-2"><code class="bg-slate-100 border-slate-300 rounded px-4 py-2"><%= @feed.token %>@<%= request.host %></code></pre>
+      <pre class="my-2"><code class="bg-slate-100 border-slate-300 rounded px-4 py-2"><%= @feed.token %>@<%= request.domain %></code></pre>
```

Comment:
TIL!

Given the existence of "top level" things like `.co.uk` I kinda wish we could express our intention more clearly by passing `[1..-1]` or something to [`domain`](https://apidock.com/rails/ActionDispatch/Http/URL/domain).

For that matter, how does this interact with the existence of a `staging` subdomain?

Reply:
yeah, so the way that works is a global configuration of how many bits to chop off:

Reply:
oops meant to link to it. here's the global config that looks at HOST to decide how many ending TLD bits to chop off: https://github.com/indirect/feedyour.email/pull/42/files#diff-d2c69534ceb90069932d2ec9d4957bc8860e8af5fbdbf04064eb6ed635d76300R6-R7

If HOST is feedyour.email, then `subdomain` will return `foo` for `foo.feedyour.email`. If HOST is staging.feedyour.email, then `subdomain` will return `foo` for `foo.staging.feedyour.email`.

Reply:
so… if HOST doesn't include the token does that mean we actually still want `host` here?